### PR TITLE
Add new command - Dice & Coin

### PR DESCRIPTION
**Pull request by yonbergman**
## Description

Simple little utility command to Toss a coin or Throw a die to produce a random result

## Screencast
Raycast 2023-01-06 at 11 15 21
Raycast 2023-01-06 at 11 15 19
 
Raycast 2023-01-06 at 11 15 14
Raycast 2023-01-06 at 11 15 08

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/random/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "random",
+  "title": "Random",
+  "description": "Helps you pick simple random values\n- Toss a coin\n- Throw a dice",
```
Comment (pernielsentikaer) [inline extensions/random/package.json:5]:
```suggestion
  "description": "Quickly and easily make random decisions or generate a random number",
```

File: extensions/random/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "random",
+  "title": "Random",
```
Comment (pernielsentikaer) [inline extensions/random/package.json:4]:
```suggestion
  "title": "Dice & Coin",
```
Could be a better name?

File: extensions/random/package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "random",
```
Comment (pernielsentikaer) [inline extensions/random/package.json:3]:
```suggestion
  "name": "dice-and-coin",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you 😊

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by yonbergman** [comment]
hi @pernielsentikaer updated the name of extension and texts :)
Thanks for the feedback

File: extensions/dice-and-coin/src/throw-dice.tsx [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import { showToast, Toast } from "@raycast/api";
 
 export default async function Command() {
-  const toast = await showToast({ style: Toast.Style.Animated, title: "Throwing die..." });
```
Comment (yonbergman) [inline extensions/dice-and-coin/src/throw-dice.tsx]: @pernielsentikaer but die is the singular of dice https://en.wiktionary.org/wiki/die#Noun_2

Comment (pernielsentikaer) [reply extensions/dice-and-coin/src/throw-dice.tsx]: Let's die, sorry 😊

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**